{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert '4cpnue2v0ja16ryu@76.116' == True\n +  where '4cpnue2v0ja16ryu@76.116' = createRandomEmail()", "trace": "def test_login():\n    \tcreateRandomEmail()\n>   \tassert createRandomEmail() == True\nE    AssertionError: assert '4cpnue2v0ja16ryu@76.116' == True\nE     +  where '4cpnue2v0ja16ryu@76.116' = createRandomEmail()\n\ntest_000_test.py:9: AssertionError"}, "start": 1620201560694, "stop": 1620201560695, "uuid": "99e6d9f5-08d6-4da4-9d9d-32c3f662c4a7", "historyId": "3c5a24146fbb36903e83f074c345f0b6", "testCaseId": "f56867ea33ff03782562f49c2cfd9d40", "fullName": "TestCase_Pytest.test_000_test#test_login", "labels": [{"name": "parentSuite", "value": "TestCase_Pytest"}, {"name": "suite", "value": "test_000_test"}, {"name": "host", "value": "DESKTOP-DA9AGC9"}, {"name": "thread", "value": "24836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase_Pytest.test_000_test"}]}